.github/workflows/docs.yml

```diff
@@ -24,35 +24,6 @@ jobs:
 
     - name: Build documentation
       run: |
-        # SPHINXOPTS="-W --keep-going -n" is used to turn warnings into errors
+        # SPHINXOPTS="-W --keep-going" is used to turn warnings into errors
         # but keep going, so we don't fail on the first one.
-        make -C docs html SPHINXOPTS="-W --keep-going -n"
-
-    - uses: actions/upload-artifact@v2
-      with:
-        name: DocumentationHTML
-        path: docs/build/html/
-
-    # Publish built docs to gh-pages branch
-    - name: Commit documentation changes
-      # push docs only when a GitHub Release is made
-      if: github.event_name == 'release' && github.event.action == 'published'
-      run: |
-        git clone https://github.com/IMAP-Science-Operations-Center/imap_processing.git --branch gh-pages --single-branch gh-pages
-        cp -r docs/build/html/* gh-pages/
-        cd gh-pages
-        git config --local user.email "action@github.com"
-        git config --local user.name "GitHub Action"
-        git add .
-        git commit -m "Update documentation" -a || true
-        # The above command will fail if no changes were present, so we ignore that.
-
-    - name: Publish docs
-      # push docs only when a GitHub Release is made
-      if: github.event_name == 'release' && github.event.action == 'published'
-
-      uses: ad-m/github-push-action@master
-      with:
-        branch: gh-pages
-        directory: gh-pages
-        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I am guessing we are removing all of this because we don't use GitHub pages now that we are using RTD?

Reply:
Exactly, this was so you could download and the artifacts created, but much easier to just look at the read the docs built PR pages instead now that we have that set up.